portfolio/src/main/webapp/script.js

```diff
@@ -73,11 +73,11 @@ function contact() {
 
 function getComments() {
 	fetch('/data').then(response => response.json()).then((comments) => {
-		const bodyElement = document.body;
-        console.log('Comments: ', comments);
-        bodyElement.innerHTML += 'Here are all the comments: <br>';
-        for (comment of comments) {
-    		bodyElement.innerHTML += comments[index] + '<br>';
-        }
+	 	const bodyElement = document.body;
+    console.log('Comments: ', comments);
+    bodyElement.innerHTML += 'Here are all the comments: <br>';
+    for (index in comments) {
+      bodyElement.innerHTML += comments[index] + '<br>';
+    }
```

Comment:
left comments on this in the parent branch but let's resolve them there...

Reply:
yep! I believe it should be updated this time!